aiogram_dialog/api/protocols/manager.py

```diff
@@ -103,8 +103,9 @@ def find(self, widget_id) -> Optional[Any]:
         which does not require to pass manager and has only subset of methods.
         """
         raise NotImplementedError
-
-    async def reset_stack(self, remove_keyboard: bool = True) -> None:
+    
+    # Shao-mod
+    async def reset_stack(self, new_text: Optional[str], remove_keyboard: bool = True) -> None:
```

Comment:
i think new argument should be last in positional args and should have default value (`None`?)